politeiawww/wsdcrdata.go

```diff
@@ -90,9 +102,90 @@ func (w *wsDcrdata) unsubFromAddr(address string) error {
 	return nil
 }
 
-// newWSDcrdata return a new wsDcrdata context.
-func newWSDcrdata() (*wsDcrdata, error) {
-	// Init websocket client
+// subToNewBlock subscribes to dcrdata events for new blocks
+func (w *wsDcrdata) subToNewBlock() error {
+	if w.isSubscribed(newBlockSub) {
+		return errDuplicateSub
+	}
+	_, err := w.client.Subscribe(newBlockSub)
+	if err != nil {
+		return fmt.Errorf("failed to subscribe: %v", err)
```

Comment:
Should include the subscription.  Can you fix this in the other subscribe and unsubscribe functions too?
```suggestion
		return fmt.Errorf("failed to subscribe to %v: %v", newBlockSub, err)
```